HomeWork_3/Task_3.py

```diff
@@ -0,0 +1,38 @@
+'''Найти самое длинное слово в введенном предложении.
+Учтите что в предложении есть знаки препинания.'''
```

Comment:
Your solution doesn't handle punctuation marks

Reply:
line = str(input('Enter string \n'))
import string
for c in string.punctuation:
    line = line.replace(c,'')
print(line)
list_words = line.split()
LongWord = 0
for i in range(1, len(list_words)):
    if len(list_words[LongWord]) < len(list_words[i]):
        LongWord = i
LongWord = list_words[LongWord].strip(",")
print(LongWord)